pkg/operator/operator.go

```diff
@@ -43,7 +43,23 @@ const (
 
 	operatorVersionEnvName = "OPERATOR_IMAGE_VERSION"
 	operandVersionEnvName  = "OPERAND_IMAGE_VERSION"
-	operandImageEnvName    = "OPERAND_IMAGE"
+
+	driverImageEnvName              = "DRIVER_IMAGE"
+	provisionerImageEnvName         = "PROVISIONER_IMAGE"
+	attacherImageEnvName            = "ATTACHER_IMAGE"
+	resizerImageEnvName             = "RESIZER_IMAGE"
+	snapshotterImageEnvName         = "SNAPSHOTTER_IMAGE"
+	nodeDriverRegistrarImageEnvName = "NODE_DRIVER_REGISTRAR_IMAGE"
+	livenessProbeImageEnvName       = "LIVENESS_PROBE_IMAGE"
+
+	// Index of a container in assets/controller_deployment.yaml and assets/node_daemonset.yaml
+	csiDriverContainerIndex           = 0 // Both Deployment and DaemonSet
+	provisionerContainerIndex         = 1
+	attacherContainerIndex            = 2
+	resizerContainerIndex             = 3
+	snapshottterContainerIndex        = 4
+	nodeDriverRegistrarContainerIndex = 1
+	livenessProbeContainerIndex       = 2 // Only in DaemonSet
```

Comment:
We can change this later, but maybe instead of hardcoding indexes we can harcode image names and have a func to create the indexes.